assets/js/translator.js

```diff
@@ -74,7 +74,6 @@ if(modeEnabled('translation')) {
             muteLanguages();
             localizeInterface();
             translateText();
-
             autoSelectDstLang();
```

Comment:
Yes makes sense. I did not pay much attention to the inside block :/ . I'd rectify this right away!